persistence/src/main/scala/hmda/persistence/processing/HmdaFileValidator.scala

```diff
@@ -72,6 +79,16 @@ class HmdaFileValidator(submissionId: String) extends HmdaPersistentActor with L
       val validationStarted = ValidationStarted(submissionId)
       publishEvent(validationStarted)
       events(parserPersistenceId)
+        .filter(x => x.isInstanceOf[TsParsed])
+        .map(e => e.asInstanceOf[TsParsed].ts)
+        .map(ts => validateTs(ts, ctx).toEither)
+        .map {
+          case Right(ts) => ts
+          case Left(errors) => ValidationErrors(errors.list.toList)
+        }
+        .runWith(Sink.actorRef(self, NotUsed))
```

Comment:
I'm not as familiar with akka streams, so I'm testing my understanding here. I think this line is saying that once the stream is processed and all Transmittal Sheets have been validated, this actor isn't listening for the "completed" event. That's okay because line 99 of this file sends another "completed" message that we do listen for.

... is that right?